Cannot find module 'deepmerge'

### Version
3.0.0-alpha.8

### Reproduction link
[https://codesandbox.io/s/vue](https://codesandbox.io/s/vue)

### Steps to reproduce
vue  create app

### What is expected?
The project can be built normally

### What is actually happening?
Error: Cannot find module 'deepmerge'
![image](https://user-images.githubusercontent.com/15010463/39390878-d80e2952-4acd-11e8-8af8-df4dc345ea72.png)

---
@vue/cli packjson lacks dependency 'deepmerge'

<!-- generated by vue-issues. DO NOT REMOVE -->

--- a/packages/@vue/cli/package.json
+++ b/packages/@vue/cli/package.json
@@ -32,6 +32,7 @@
     "commander": "^2.12.2",
     "debug": "^3.1.0",
     "download-git-repo": "^1.0.2",
+    "deepmerge": "^2.0.1",
     "ejs": "^2.5.7",
     "execa": "^0.8.0",
     "globby": "^8.0.1",

